### ISSUE TITLE
[Bug]: Text alignment right of fluent-text-field doesn't work properly (not orientation rtl)

### ISSUE THREAD

**Issue by gomineta** [microsoft/fluentui#27917]
### Library

Web Components (@fluentui/web-components)

### System Info

```shell
N/A
```

### Are you reporting Accessibility issue?

None

### Reproduction

https://codepen.io/gomineta/pen/xxyQqzL

## Actual Behavior
Using a css rule over a selector of a fluent-text-field or fluent-number-field do not align the text to the right.

## Expected Behavior
The alignment of the text inside a fluent-text-field or fluent-number-field should align to the right.

### Requested priority

Normal

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by chrisdholt** [comment]
Thanks for the issue @gomineta - this is working as expected so I'm going to close this, but I've got an update to your example which can illustrate how to apply styles in this specific case. The styles you are trying to apply to the host element itself need to be applied within the component's shadow root. For convenience in these situations we've currently exposed CSS Parts which allow you to apply styles to specific internals of the Web Component. Here's an example of how to target the `control` part in order to apply the styles to the shadowed input: https://stackblitz.com/edit/web-platform-ajrjov?file=index.html. As a note, I've also updated this to use the value `end` as that will localize for both LTR and RTL as expected.
